src/dataframe/dataframe.jl

```diff
@@ -894,20 +917,21 @@ function insertcols!(df::DataFrame, col::ColumnIndex, name_cols::Pair{Symbol, <:
 end
 
 insertcols!(df::DataFrame, col::ColumnIndex, name_cols::Pair{<:AbstractString, <:Any}...;
-                     makeunique::Bool=false, copycols::Bool=true) =
+            after::Bool=false, makeunique::Bool=false, copycols::Bool=true) =
     insertcols!(df, col, (Symbol(n) => v for (n, v) in name_cols)...,
-                makeunique=makeunique, copycols=copycols)
+                after=after, makeunique=makeunique, copycols=copycols)
 
 insertcols!(df::DataFrame, name_cols::Pair{Symbol, <:Any}...;
-            makeunique::Bool=false, copycols::Bool=true) =
-    insertcols!(df, ncol(df)+1, name_cols..., makeunique=makeunique, copycols=copycols)
+            after::Bool=false, makeunique::Bool=false, copycols::Bool=true) =
+    insertcols!(df, ncol(df)+1, name_cols..., after=after,
+                makeunique=makeunique, copycols=copycols)
 
 insertcols!(df::DataFrame, name_cols::Pair{<:AbstractString, <:Any}...;
-            makeunique::Bool=false, copycols::Bool=true) =
+            after::Bool=false, makeunique::Bool=false, copycols::Bool=true) =
     insertcols!(df, (Symbol(n) => v for (n, v) in name_cols)...,
-                makeunique=makeunique, copycols=copycols)
+                after=after, makeunique=makeunique, copycols=copycols)
 
-function insertcols!(df::DataFrame, col::Int=ncol(df)+1; makeunique::Bool=false, name_cols...)
+function insertcols!(df::DataFrame, col::Int=ncol(df)+1; after::Bool=false, makeunique::Bool=false, name_cols...)
```

Comment:
this is not needed - this method was kept only to throw an error for a deprecated functionality.